installer/bundle/bundle_skel.sh

```diff
@@ -1444,6 +1452,13 @@ case "$installMode" in
                 OSS_BUNDLE=`basename $i -oss-test.sh`
                 [ ! -f oss-kits/${OSS_BUNDLE}-cimprov-*.sh ] && continue
 
+                if [ "$OSS_BUNDLE" = "docker" ]  && [ "$skipDockerProviderInstall" = "true" ]; then
+                   echo "$OSS_BUNDLE provider package installation skipped since --skip-docker-provider-install flag is set.
```

Comment:
IIRC this will cause new lines and formatting like
```
blah blah blah
                   blah blah blah blah
                   blah blah blah blah
```

So best to just oneline it and let the console wrap it.

Reply:
fixed formatting issue.